templates/ts-apollo-postgres-backend/model/datamodel.graphql

```diff
@@ -4,14 +4,19 @@ type Note {
   title: String!
   description: String
   """
-  @oneToMany(field: 'note')
+  @db(type: 'json')
   """
-  comments: [Comment]!
+  comment: Comment
 }
 
-""" @model """
 type Comment {
   id: ID!
-  text: String
-  description: String
+  text: String!
+  description: String!
+  test: Test!
+  test2: [Test!]!
+}
+
+type Test {
+  hello: String
```

Comment:
Maybe we could add an [integration test](https://github.com/aerogear/graphback/tree/9622f7373cb59f3926b6a538ea4424de29af7d4e/integration) and revert the change on this template file?

Reply:
Reverted file now, thanks. Have begun the process of adding integration test cases but not sure if I will get it complete in time (off tomorrow).